src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+ensure that the entire image is always visible, and so the opposite of `cover`:
+where it takes the largest boundary (in our example above this is width), and
+resize the image to maintain its intrinsic aspect-ratio while fitting into the
+space. The `object-fit: none` case shows the image cropped in it's center
+(default object position) at its natural size.
+
+`object-fit: cover` tends to work in most situations to ensure a nice uniform
+interface when dealing with images of varying dimensions, however, you lose data
+this way (the image is cropped at its longest edges). 
+
+If these details are important (for example, when working with a flat lay of
+beauty products), cropping important content is not acceptable. So the ideal
+scenario would be responsive images of varying sizes that fit the UI space
+without cropping.
+
+## Maintaining aspect ratio with `padding-top`
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspectratio-current.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Using `padding-top` to set a 1:1 aspect ratio on post preview images within a carousel.
+  </figcaption>
+</figure>
+
+In order to make these more responsive, we can use aspect-ratio. This allows for
+us to set a specific ratio size and base the rest of the media on an individual
+axis (height or width).
+
+A currently well-accepted cross-browser solution for maintaining aspect ratio
+based on an image's width is known as the “Padding-Top Hack". This solution
+requires a parent container and an absolutely placed child container. One would
+then calculate the aspect ratio as a percentage to set as the `padding-top`. For
+example:
+
+1:1 aspect ratio = 1 / 1 = 1 = `padding-top: 100%` 4:3 aspect ratio = 3 / 4 =
+0.75 = `padding-top: 75%` 3:2 aspect ratio = 2 / 3 = 0.66666 = `padding-top:
+66.67%` 16:9 aspect ratio = 9 / 16 = 0.5625 = `padding-top: 56.25%`
+
+Now that we have identified the aspect ratio value, we can apply that to our
+parent container. Consider the following example:
+
+```html
+<div class="container">
+  <img class="media" src="..." alt="">
+</div>
+```
+
+We could then write the following CSS:
+
+```css
+.container {
+  position: relative;
+  width: 100%;
+  padding-top: 56.25%; /* 16:9 Aspect Ratio */
+}
+
+.media {
+  position: absolute;
+  top: 0;
+}
+```
+
+## Maintaining aspect ratio with `aspect-ratio`
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="aspectratio-future.jpg" alt="">
+  <figcaption class="w-figcaption">
+    Using `aspect-ratio` to set a 1:1 aspect ratio on post preview images within a carousel.
+  </figcaption>
+</figure>
+
+Unfortunately, calculating these `padding-top` values is not very intuitive, and
+requires some additional overhead and positioning. With the new intrinsic
+`aspect-ratio` [CSS property](https://www.w3.org/TR/css-sizing-4/#aspect-ratio),
+coming to Chromium 88, Safari Technical Preview 118, and Firefox __, the
```

Comment:
Links for Chromium, Safari, and Firefox plz

Reply:
Can you also update MDN's browser compat table or at least file an issue? https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#browser_compatibility